Parsedown.php

```diff
@@ -469,6 +469,20 @@ protected function blockFencedCode($Line)
 
         if ($infostring !== '')
         {
+            /**
+             * https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#classes
+             * Every HTML element may have a class attribute specified.
+             * The attribute, if specified, must have a value that is a set
+             * of space-separated tokens representing the various classes
+             * that the element belongs to.
+             * [...]
+             * The space characters, for the purposes of this specification,
+             * are U+0020 SPACE, U+0009 CHARACTER TABULATION (tab),
+             * U+000A LINE FEED (LF), U+000C FORM FEED (FF), and
+             * U+000D CARRIAGE RETURN (CR).
+             */
+            $language = substr($infostring, 0, strcspn($infostring, " \t\n\f\r"));
```

Comment:
```suggestion
            $infostring = substr($infostring, 0, strcspn($infostring, " \t\n\f\r"));
```

Reply:
Oops, meant to rename the use-site too :) https://github.com/erusev/parsedown/commit/8fd5464c46ddb778e65a53d60f72e06f7f7e5329